integreat_cms/core/management/commands/repair_tree.py

```diff
@@ -132,8 +132,11 @@ def check_for_orphans(self, tree_id):
             for orphan in orphans:
                 self.stdout.write(self.bold(f"Page {orphan.id}:"))
                 self.print_error(
-                    f"\tparent_id: {orphan.parent_id}\n"
-                    f"\tparent.tree_id: {orphan.parent.tree_id}"
+                    f"\tparent_id: {orphan.parent_id}"(
+                        f"\n\tparent.tree_id: {orphan.parent.tree_id}"
+                        if orphan.tree
+                        else f""
+                    )
```

Comment:
```suggestion
                self.print_error(f"\tparent_id: {orphan.parent_id}")
                if orphan.parent_id:
                    self.print_error(f"\tparent.tree_id: {orphan.parent.tree_id}")
```
Thanks for the quick fix! I guess this would be better readable?